apps/in_hand_scanner/src/icp.cpp

```diff
@@ -178,7 +178,7 @@ pcl::ihs::ICP::findTransformation (const MeshConstPtr&              mesh_model,
 
   // Convergence and registration failure
   float current_fitness  = 0.f;
-  float previous_fitness = std::numeric_limits <float>::max ();
+  float previous_fitness;
```

Comment:
Please reduce the scope of this variable instead of have a uninitialized variable.